source/common/version/BUILD

```diff
@@ -59,7 +59,7 @@ envoy_cc_library(
     name = "version_lib",
     srcs = ["version.cc"],
     copts = envoy_select_boringssl(
-        ["-DENVOY_SSL_VERSION=\\\"BoringSSL-FIPS\\\""],
+        ["-DENVOY_SSL_FIPS -DENVOY_SSL_VERSION=\\\"BoringSSL-FIPS\\\""],
```

Comment:
This should be a list of 2 parameters:
```
["-DENVOY_SSL_FIPS", "-DENVOY_SSL_VERSION=\\\"BoringSSL-FIPS\\\""],
```

Reply:
@PiotrSikora Initially I tried with that but it was failing.
```
<command line>:11:23: error: ISO C99 requires whitespace after the macro name [-Werror,-Wc99-extensions]
#define ENVOY_SSL_FIPS, 1
```

Reply:
Previously (https://github.com/envoyproxy/envoy/pull/16140/commits/8bd10d25ee020c68944b8af427a1f0b29fea4130), you had it written as a single parameter (notice quotes):
```
["-DENVOY_SSL_FIPS, -DENVOY_SSL_VERSION=\\\"BoringSSL-FIPS\\\""],
```
whereas it should be:
```
["-DENVOY_SSL_FIPS", "-DENVOY_SSL_VERSION=\\\"BoringSSL-FIPS\\\""],
```

Reply:
Also, while you're making changes, please add it at the end, after `ENVOY_SSL_VERSION`, since then it's easier to compare it with the non-FIPS option below.